src/test/java/ParserTest.java

```diff
@@ -0,0 +1,27 @@
+import org.junit.jupiter.api.Test;
```

Comment:
In our Java coding standard, a class should at least be a part of a package. It is better to put every class in their own package.